app/scripts/views/mailer/resetPassword.ejs

```diff
@@ -0,0 +1,18 @@
+Pepyatka password reset
```

Comment:
This is a **huge** task to be implemented in this PR. Though I'll create a new story.